docs/guides/real-time/chat/javascript.md

```diff
@@ -298,8 +298,8 @@ This guide covers authentication, item creation, and subscription using WebSocke
 							type: 'subscribe',
 							collection: 'messages',
 							query: {
-								fields: ['*', 'user_created.first_name'],
-								sort: 'date_created',
+                fields: ['*', {'user_created': ['first_name']}],
+				        sort: ['date_created'],
```

Comment:
I believe these 2 lines requires more indentations (so that they'll be nested "inside" `query`)

Reply:
I'll lint & format the code blocks as soon as this PR is considered as ready to be merged.
@estheragbaje just let me know when that's the case

Reply:
Will do, gracias!